src/file/URLSearchParamsRe.re

```diff
@@ -0,0 +1,15 @@
+[@bs.deriving abstract]
+type t;
+
+[@bs.new] external make: string => t = "URLSearchParams";
+[@bs.send] external append: (t, 'a, 'b) => unit = "append";
+[@bs.send] external delete: (t, 'a) => unit = "delete";
+[@bs.send] external entries: t => Js.Array.array_like('a) = "entries";
+[@bs.send] external get: (t, 'a) => option('b) = "get";
+[@bs.send] external getAll: (t, 'a) => Js.Array.array_like('a) = "getAll";
+[@bs.send] external has: (t, 'a) => bool = "has";
+[@bs.send] external keys: t => array('a) = "keys";
```

Comment:
This should be an `array_like` (or `iterator`, technically, but that concept does not exist yet in BuckleScript)